tests/utils/fetch.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { appendUrlParam } from '@common/api/fetch'
+
+test('Append url correctly', () => {
+  const rootApiUrl = "https://stacks-node-api.testnet.stacks.co/extended/v1"
+  const addressTxUrl = rootApiUrl + '/address/STM6S3AESTK9NAYE3Z7RS00T11ER8JJCDNTKG711/transactions?limit=50'
+  expect(appendUrlParam('extended/v1', 'unanchored', 'true')).toEqual('extended/v1?unanchored=true')
+  expect(appendUrlParam('', 'unanchored', 'true')).toEqual('?unanchored=true')
+  expect(appendUrlParam(addressTxUrl, 'unanchored', 'true')).toEqual(addressTxUrl + '&unanchored=true')
+});
```

Comment:
```suggestion
describe(appendUrlParam.name, () => {
  test('it appends url correctly', () => {
    const rootApiUrl = "https://stacks-node-api.testnet.stacks.co/extended/v1"
    const addressTxUrl = rootApiUrl + '/address/STM6S3AESTK9NAYE3Z7RS00T11ER8JJCDNTKG711/transactions?limit=50'
    expect(appendUrlParam('extended/v1', 'unanchored', 'true')).toEqual('extended/v1?unanchored=true')
    expect(appendUrlParam('', 'unanchored', 'true')).toEqual('?unanchored=true')
    expect(appendUrlParam(addressTxUrl, 'unanchored', 'true')).toEqual(addressTxUrl + '&unanchored=true')
  });
});
```

Preference, but I typically always wrap in a describe block, and use the global to create a readable sentence

`test('that it')...`
`it('should...`

Reply:
Yes this is better